pkg/resource/aws/aws_route53_record_test.go

```diff
@@ -1,29 +1,37 @@
-package aws_test
+package aws
 
 import (
 	"testing"
 
-	"github.com/cloudskiff/driftctl/test/acceptance"
+	"github.com/aws/aws-sdk-go/aws"
 )
 
-func TestAcc_AwsRoute53Record_WithFQDNAsId(t *testing.T) {
```

Comment:
This test should not be modified, it should remain an acceptance test

Reply:
I think i've undone modification on this test.